### DISCUSSION TITLE
Taskbar Item Width?

### DISCUSSION THREAD

**Discussion by turtlepod** [valinet/ExplorerPatcher#557] [Q&A] [answered]
How to set taskbar item width?
Is there a settings to set the width to be fixed?

In my computer it seems to try to use the maximum width.
Thanks

**Comment by valinet** [comment, marked as answer]
Taskbar button width is internally determined by the taskbar code in explorer. No setting is exposed in any GUI.

If the width of the button is abnormally big, try rebooting the computer, and then, if still not solved, maybe try this: https://www.tenforums.com/tutorials/104754-change-width-taskbar-buttons-windows.html

**Reply by turtlepod** [reply]
Thank you @valinet 
The `MinWidth` entry is actually not available in my installation.
I simply added it to the registry with the value `38`, and it now showing correctly.

:yay:
